linode/manager
# fix: [M3-7249] - Linodes Landing Glitchy Render Behavior

## Description 📝

Are you sick and tried of the Linodes Landing page flashing at you? If so, this PR is for you!

## Changes  🔄
- Use preferences directly to control the SideNav (no longer uses `PreferenceToggle`)
- Gives the `PreferenceToggle` the correct default state value

## Preview 📷

| Before  | After   |
| ------- | ------- |
| <video src="https://github.com/linode/manager/assets/115251059/1f80c220-1566-43d5-9901-f52f7451e82d" /> | <video src="https://github.com/linode/manager/assets/115251059/59a81565-318f-403a-a43f-d482e2597764" /> |

## How to test 🧪

### Prerequisites
- Have at least 1 Linode on your account

### Reproduction steps
- Go to `cloud.linode.com` and go to the Linodes Landing page
- Click the Akamai logo many times
- Observe the Linodes table flashing when you click the Akamai logo

### Verification steps 
- Go to `http://localhost:3000/linodes`
- Click the Akamai logo many times
- Observe the Linodes table **does not** flash at you and the UI stays the same

## As an Author I have considered 🤔
